dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -251,10 +252,10 @@ a.header_status_bar:hover {
 
 .main {
   padding: 10px;
-  a:link {font-family: $gotham-extra-bold; font-weight: normal; color:$dark_color;}
-  a:visited {font-family: $gotham-extra-bold; font-weight: normal; color:$dark_color;}
-  a:hover {font-family: $gotham-extra-bold; font-weight: normal; color:$dark_color; background-color: transparent;   text-decoration: underline; }  /* mouse over link */
-  a:active {font-family: $gotham-extra-bold; font-weight: normal; color:$dark_color; background-color: transparent;}  /* selected link */
+  a:link {font-family: $gotham-extra-bold; font-weight: normal; color: $dark_color;}
+  a:visited {font-family: $gotham-extra-bold; font-weight: normal; color: $dark_color;}
+  a:hover {font-family: $gotham-extra-bold; font-weight: normal; color: $dark_color; background-color: transparent;   text-decoration: underline; }  /* mouse over link */
+  a:active {font-family: $gotham-extra-bold; font-weight: normal; color: $dark_color; background-color: transparent;}  /* selected link */
```

Comment:
It seems reasonable to break these out to separate lines too.  Like, they're special, but not _that_ special :)